e2e/commonContext.go

```diff
@@ -215,7 +214,7 @@ var (
 				var r apis.Response
 				err = json.Unmarshal(result, &r)
 				gomega.Expect(http.StatusOK).Should(gomega.Equal(r.Code))
-				output := fmt.Sprintf("Create env succeed, current env is " + envMeta.Name + " namespace is " + envMeta.Namespace)
```

Comment:
I think the former expected output is more specific.

Reply:
Now the create an env interface does not provide the ability to switch env.

Reply:
I got it. @wonderflow is it okay with your initial design?